### PR TITLE
Feature/payments/ci

### PR DESCRIPTION
**Pull request by jphillips2121**
- Added dependencies to POM.xml 
- Added routes.yaml file 
- Change start script permission so it can run

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -33,12 +38,22 @@
...
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-devtools</artifactId>
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <groupId>uk.gov.companieshouse</groupId>
+            <artifactId>structured-logging</artifactId>
+            <version>${structured-logging.version}</version>
+            <scope>compile</scope>
```
Comment (ChrisJBurns) [inline pom.xml]: no need to specify the compile scope as its default

File: pom.xml [hunk 2]
```diff
@@ -21,7 +21,12 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+
+        <!-- Java -->
         <java.version>1.8</java.version>
+
+        <!-- Dependency Versions -->
+        <structured-logging.version>1.2.0-rc1</structured-logging.version>
```
Comment (ChrisJBurns) [inline pom.xml]: i believe there is a new version of the structured logging

File: routes.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+app_name: payments.web.ch.gov.uk
+group: web
+# XXX Make sure all service with /transaction/ based routes have a weight LOWER than 1000
+weight: 900
+routes:
+   1: /transactions/(.*)/payments/(.*)/.*
```
Comment (BSaunders95) [inline routes.yaml]: Should this be `transaction` rather than `transactions`?

Comment (jphillips2121) [reply routes.yaml]: Discussed this and we're not using transactions at all so will remove this.

Comment (kevbradwick) [reply routes.yaml]: a routes yaml may still be needed - in previous discussions we've said that the payment platform will be on a different domain. not sure how routes should be configured but i'm sure @macreman will be able to help with that.

Comment (justinblandford1) [reply routes.yaml]: Yes we still need the routes yams from what we understand, its just that there's knowledge bleed in the URL as the payment service should be agnostic of the transaction.  We will push this change up through CI again on Ernie and retest before opening this up for review again.

**Pull request merged**